User-Service/Data/Data Access/UserService.cs

```
@@ -0,0 +1,35 @@
+﻿using User_Service.Data;
+using User_Service.Model;
+
+namespace User_Service
+{
+    public class UserService
+    {
+        private readonly AppDbContext context;
+        public UserService(AppDbContext context)
+        {
+            this.context = context;
+        }
+      public String GetUserGoogleIdbyUserID(Guid userId)
+
+        {
+            // Returns the googleID of the user by the gived Guid userID
+
+            try
+            {
+
+                string googleID = context.Users.Where(x => x.UserId == userId).Select(x => x.GoogleId).ToString();
+                return googleID;
+
+            }
+            catch (InvalidCastException e)
+            {
+                // recover from exception
+                return "User googleID not found";
```

Returning a result like this will cause extra checking. Rather bubble up the exception or return a null object